graalvm/build.gradle.kts

```diff
@@ -19,21 +19,54 @@ dependencies {
     implementation("ch.qos.logback:logback-classic:1.4.6")
     implementation("io.ktor:ktor-server-core-jvm")
     implementation("io.ktor:ktor-server-cio-jvm")
+
+    testImplementation("io.ktor:ktor-server-test-host-jvm")
+    testImplementation("org.jetbrains.kotlin:kotlin-test")
 }
 
 graalvmNative {
     binaries {
+
         named("main") {
             fallback.set(false)
             verbose.set(true)
 
-            buildArgs.add("--initialize-at-build-time=io.ktor,kotlin")
+            buildArgs.add("--initialize-at-build-time=ch.qos.logback")
+            buildArgs.add("--initialize-at-build-time=io.ktor.kotlin")
+            buildArgs.add("--initialize-at-run-time=org.slf4j.LoggerFactory")
+            buildArgs.add("--initialize-at-run-time=org.slf4j.impl.StaticLoggerBinder")
+            buildArgs.add("-H:+InstallExitHandlers")
+            buildArgs.add("-H:+ReportUnsupportedElementsAtRuntime")
+            buildArgs.add("-H:+ReportExceptionStackTraces")
+
+            imageName.set("graalvm-server")
+        }
+
+        named("test"){
+            fallback.set(false)
+            verbose.set(true)
+
+            buildArgs.add("--initialize-at-build-time=ch.qos.logback")
+            buildArgs.add("--initialize-at-build-time=io.ktor.kotlin")
```

Comment:
Could you log an issue about problems without this argument?

Reply:
Logged an issue [KTOR-6074](https://youtrack.jetbrains.com/issue/KTOR-6074/GraalVM-native-image-cant-compile-logback-dependencies)

Reply:
Also logged an issue for logger: [KTOR-6075](https://youtrack.jetbrains.com/issue/KTOR-6075/GraalVM-native-image-cant-compile-because-of-static-logger-initialization-in-RoutingKt)